tests/sentry/integrations/slack/notifications/test_deploy.py

```diff
@@ -9,7 +9,7 @@
 from sentry.testutils.helpers.slack import get_attachment, send_notification
 
 
-class SlackUnassignedNotificationTest(SlackActivityNotificationTest):
+class SlackDeployNotificationTest(SlackActivityNotificationTest):
```

Comment:
Good catch but these 7 test file changes don't seem related to the rest of the PR.